_Week8/group1/README.md

```diff
@@ -0,0 +1,32 @@
+
+The parameter that we vary is the number of strata in our sample. In this case, the number of strata corresponds to the number of states that we are sampling from.  We generate the state-level fixed effects using a chi squared distribution— the rationale behind this is purely because we wanted to practice using a variety of different distributions in the randomization processes of our assignment. Then, we uniformly randomize the number of universities within each state, and generate the university-level fixed effects using a normal distribution. Lastly, we uniformly randomize the number of students per university.
+
+Once we define the relevant strata and fixed effects of our analysis, we continue on to generate the individual characteristics of the students. We generate variables for the parents' level of education (uniformly randomized), SAT score (randomized using a normal distribution), and AP credit (randomized using a Poisson distribution). Then, in order to generate the GPA variable that gets put in the regression for salary later in the program, we use the characteristics discussed above as well as a normally distributed randomization term. In order to generate university major-fixed effects, which also gets put into the regression for salary, we use uniform randomization. We define a local which represents the true coefficient on GPA, which we set as 5000. With these inputs, we then create the generative model for post-graduate salary on line 66, which is our main DGP. 
+
+From line 69 to line 84, we are generating categorical variables based on the salary variable and the parents' education variables. In the next section off code, we add an individual ID variable which is a concatenation of the state, university, and student variables with a leading 0. By defining the individual ID in this way, the variable tells us identifying information about each individual, which is more useful than simple generating a randomized individual ID variable. 
+
+Lastly, we run the regression of salary on GPA, undergraduate major, state in which their university is located, and university attended, in order to re-estimate the coefficient on GPA. The program ends by returning the beta coefficient and standard error on GPA, as well as the total sample size. 
+
+
+## results.do
+
+In the results do file, we run an example of this program in practice. We create a loop over the parameter index (number of state strata we sampled from). We then create a nested loop within the parameter loop in order to set the number of samples we draw. In this case, the outer loop indicates that we loop over 20 states, and the inner loop indicates that we will draw 5 samples. In the first iteration of the outer loop, 5 samples will be drawn from 1 state; in the 10th iteration. Of the outer loop, 5 samples will be drawn from a collection of 10 states (thus, the number of strata used to pull the samples is cumulative). 
```

Comment:
This approach seems a little unclear to me, both in function and purpose. Why are you varying the number of states? Why only draw 5 samples?